tasty-hedgehog.cabal

```diff
@@ -24,7 +24,7 @@ library
   build-depends:       base >= 4.8 && <4.13
                      , tagged >= 0.8 && < 0.9
                      , tasty >= 0.11 && < 1.3
-                     , hedgehog >= 0.5 && < 0.7
+                     , hedgehog >= 1.0 && < 2.0
```

Comment:
I think these bounds are a bit extreme, you probably meant `>= 1.0 && < 1.1`

Reply:
Right, this is fixed now.